charts/rancher-monitoring/101.1.0+up40.1.2/charts/kube-state-metrics/templates/servicemonitor.yaml

```diff
@@ -35,6 +35,18 @@ spec:
     {{- if .Values.prometheus.monitor.metricRelabelings }}
       metricRelabelings:
         {{- toYaml .Values.prometheus.monitor.metricRelabelings | nindent 8 }}
+        {{ if .Values.global.cattle.clusterId }}
+        - sourceLabels: [__address__]
+          targetLabel: cluster_id
+          replacement: {{ .Values.global.cattle.clusterId }}
+         {{- end }}
+     {{ else }}
+       {{ if .Values.global.cattle.clusterId }}
+      metricRelabelings:
+        - sourceLabels: [__address__]
+          targetLabel: cluster_id
+          replacement: {{ .Values.global.cattle.clusterId }}
+       {{- end }}
```

Comment:
same comment as above; move the `metricsRelabeling` outside the `if` statement and you can avoid repeating code.